### PR TITLE
Residue properties

### PR DESCRIPTION
**Pull request by frodofine**
Maybe this is better as an atom property, but the chainid is generally regarded as a residue property (for example, how the MMTF data structure is ordered).

### CODE REVIEW

**Comment by Luthaf** [comment]
This looks good to me! Having Residues properties as well as atomic and frame properties make complete sense.

Could you add the corresponding C API functions (`chfl_residue_set_property`, `chfl_residue_get_property`) ?

**Comment by frodofine** [comment]
Sure thing.

File: include/chemfiles/capi/atom.h [hunk 1]
```diff
@@ -188,7 +188,7 @@ CHFL_EXPORT chfl_status chfl_atom_set_property(
 
 /// Get a property with the given `name` in this `atom`.
 ///
-/// This function returns `NULL` is no property exist with the given name.
+/// This function returns `NULL` if no property exists with the given name.
```
Comment (Luthaf) [inline include/chemfiles/capi/atom.h]: thanks! I am not a native speaker, so there might be more of them hiding around

**Comment by Luthaf** [comment]
Just though of this: could you also add you changes to the CHANGELOG.md ? Both new formats (SDF and MMTF) and this ?

File: include/chemfiles/capi/atom.h (hunk 1 above)
Comment (frodofine) [reply include/chemfiles/capi/atom.h]: Not a problem, I'll fix 'em as I see 'em.

**Comment by Luthaf** [comment]
Thanks! I'll wait for Travis and Appveyor tests to pass, and then I'll merge this.

**Pull request merged**